snapshot_manager/tests/github_util_test.py

```diff
@@ -1,125 +1,749 @@
-        cfg.datetime = datetime.datetime(year=2024, month=2, day=27)
-        self.assertEqual("20240227", cfg.yyyymmdd)
-
-        try:
-            gh = github_util.GithubClient(config=cfg)
-        except github_util.MissingToken:
-            pytest.skip(
-                "Skip test because this execution doesn't have access to a Github token"
+
+def test_minimize_comment_as_outdated__unssupprted_type(github_client_fxt):
+    obj = 0.2
+    with pytest.raises(expected_exception=ValueError) as ex:
+        github_client_fxt.minimize_comment_as_outdated(object=obj)
+    assert str(ex.value) == f"invalid comment object passed: {obj}"
+
+
+def test_unminimize_comment__with_issue_comment(github_client_fxt):
+    gh = github_client_fxt
+    node_id = "12345"
+    issue_comment_mock = mock.MagicMock(
+        spec=github.IssueComment.IssueComment
+    )  # Spec parameter is important to pass isinstance()
+    with mock.patch.dict(
+        issue_comment_mock.raw_data, values={"node_id": node_id}, create=True
+    ) as node_id_mock:
+        node_id_mock.__getitem__.return_value = node_id
+        with mock.patch.object(
+            gh.gql, "run_from_file", autospec=True
+        ) as run_from_file_mock:
+            run_from_file_mock.return_value = {
+                "data": {
+                    "unminimizeComment": {"unminimizedComment": {"isMinimized": False}}
+                }
+            }
+
+            actual = gh.unminimize_comment(object=issue_comment_mock)
+
+            assert actual == True
+            run_from_file_mock.assert_called_once_with(
+                variables={"id": node_id},
+                filename=gh.abspath("graphql/unminimize_comment.gql"),
             )
 
-        issue = gh.get_todays_github_issue(
-            strategy="big-merge", github_repo="fedora-llvm-team/llvm-snapshots"
+
+def test_unminimize_comment__with_str(github_client_fxt):
+    gh = github_client_fxt
+    node_id = "12345"
+    with mock.patch.object(
+        gh.gql, "run_from_file", autospec=True
+    ) as run_from_file_mock:
+        run_from_file_mock.return_value = {
+            "data": {
+                "unminimizeComment": {"unminimizedComment": {"isMinimized": False}}
+            }
+        }
+
+        actual = gh.unminimize_comment(object=node_id)
+
+        assert actual == True
+        run_from_file_mock.assert_called_once_with(
+            variables={"id": node_id},
+            filename=gh.abspath("graphql/unminimize_comment.gql"),
         )
 
-        self.assertIsNotNone(issue)
-        self.assertEqual(287, issue.number)
 
-        # Example: Get issue when in fact, no issue was created (should be always True for the day after tomorrow)
-        cfg.datetime = datetime.datetime.now(datetime.UTC) + datetime.timedelta(days=2)
-        issue = gh.get_todays_github_issue(
-            strategy="big-merge", github_repo="fedora-llvm-team/llvm-snapshots"
+def test_unminimize_comment__unssupprted_type(github_client_fxt):
```

Comment:
Another typo in the function name `unssupprted`.

Reply:
Fixed in a705220